### PR TITLE
New Feature

### PR DESCRIPTION
**Pull request by axdme**
Add a new file  with a function , which calculates nth fibonacci number.

### CODE REVIEW

File: nth_fibonacci_number.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    numbers = [0] * (n + 1)
+    numbers[1] = 1
+    for i in range(2, n + 1):
+        numbers[i] = numbers[i-2] + numbers[i-1]
+    
+    return numbers[-1]
+
+assert nth_fibonacci(0) == 0
+assert nth_fibonacci(5) == 5
+assert nth_fibonacci(13) == 233
```
Comment (mateuszk098) [inline nth_fibonacci_number.py:22]: Always leave a blank line at the end.

Comment (axdme) [reply nth_fibonacci_number.py:22]: Sure

**Review by mateuszk098 — APPROVED** [review]
Okay

**Comment by mateuszk098** [comment]
We can merge that.

**Pull request merged**